### PR TITLE
Print logging handlers to stderr on setup.py failure

### PR DESCRIPTION
**Pull request by gordonmessmer**

### CODE REVIEW

**Comment by gordonmessmer** [comment]
Alternate implementation does not access "logging" module internals.

**Comment by hroncok** [comment]
I like this one better.

**Comment by gordonmessmer** [comment]
Is that approval?

File: pyp2rpm/metadata_extractors.py [hunk 1]
```diff
@@ -353,7 +353,10 @@ def _get_metadata(self, temp_dir):
                     logger.error("The JSON was: {0}".format(e.doc))
                 self.unsupported_version = current_version
         else:
-            sys.stdout.write("Failed to extract data from setup.py script.\n")
+            import pyp2rpm.logger
```
Comment (hroncok) [inline pyp2rpm/metadata_extractors.py]: I don't particularly like this import here.

**Review by hroncok — CHANGES_REQUESTED** [review]

**Pull request merged**